sql_export/tests/__init__.py

```diff
@@ -0,0 +1,2 @@
+# -*- coding: utf-8 -*-
+from . import test_sql_query
```

Comment:
Your tests are never launched by odoo .
You need to put your test class into a checks list
```python
checks = [
    test_sql_query
]
```